docs/ManualDeploymentGuideScalarDLAuditorOnAKS.md

```diff
@@ -1,125 +1,98 @@
+## Step 7. Create a bastion server for ScalarDL Auditor
 
-## Step 5. Create an AKS cluster for Scalar DL Auditor
-
-Create an AKS cluster for the deployment of Scalar DL Auditor. Please refer to the following document for more details.  
-
-* [Create an AKS cluster for Scalar Products]() // TODO: Create a new document
-
-## Step 6. Create a bastion server for Scalar DL Ledger
-
-For executing some tools to deploy and manage Scalar DL Ledger on AKS, you need to prepare a bastion server in the same VNet of the AKS cluster you created in **Step 4**. Please refer to the following document for more details.  
-
-* [Create a bastion server]() // TODO: Create a new document
-
-## Step 7. Create a bastion server for Scalar DL Auditor
-
-For executing some tools to deploy and manage Scalar DL Auditor on AKS, you need to prepare a bastion server in the same VNet of the AKS cluster you created in **Step 5**. Please refer to the following document for more details.  
-
-* [Create a bastion server]() // TODO: Create a new document
+For executing some tools to deploy and manage ScalarDL Auditor on AKS, you need to prepare a bastion server in the same VNet of the AKS cluster you created in **Step 3**. See [Create a bastion server](./CreateBastionServer.md) for more details.
 
 ## Step 8. Create network peering between two AKS clusters
 
-To make Scalar DL work properly, Scalar DL Ledger and Scalar DL Auditor need to connect with each other. So, you need to connect two VNets using [Virtual Network Peering](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview). Please refer to the following document for more details.  
-
-* [Create network peering for Scalar DL Auditor mode]() // TODO: Create a new document
-
-## Step 9. Prepare a custom values file of Helm Chart of Scalar DL Ledger
-
-You need to configure a custom values file for the Helm Chart of Scalar DL Ledger and Scalar DL Schema Loader (for Ledger) based on your environment (e.g., access information of the database you created in **Step 2**). Please refer to the following document for more details.  
-
-## Step 10. Deploy Scalar DL Ledger using Scalar Helm Chart
-
-Deploy Scalar DL Ledger on your AKS cluster using Scalar Helm Chart. Please refer to the following document for more details.  
-
-* [Deploy Scalar Products using Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
+To make ScalarDL work properly, ScalarDL Ledger and ScalarDL Auditor need to connect with each other. You need to connect two VNets using [Virtual Network Peering](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview). See [Configure network peering for ScalarDL Auditor mode](./NetworkPeeringForScalarDLAuditor.md) fro more details.
 
-## Step 11. Prepare a custom values file of Helm Chart of Scalar DL Auditor
+## Step 9. Prepare a custom values file of Helm Chart of ScalarDL Ledger
```

Comment:
> Do we need to configure two custom values files in this case (one for ScalarDL Ledger and one for ScalarDL Schema Loader (for Ledger)? If so, let's use the following:

Yes. We have to create two custom values files since we use two Helm Charts (One is for ScalarDL Ledger, and the other is for ScalarDL Schema Loader).